### DISCUSSION TITLE
[Roadmap] v1.19

### DISCUSSION THREAD

**Discussion by volkov-invest** [RussianInvestments/investAPI#95] [Announcements]
Следующий номер релиза по внутренним причинам станет 1.19

- доработки песочницы: добавление проверки стакана, добавление учета ликвидности портфеля, исправление исполнения заявок по фьючерсам, приведение данных в стриминге песочницы к формату боевого контура; 
- исправление даты выплаты купонов;
- поддержка счетов с типом "инвесткопилка" в портфолио;
- улучшение скрипта закрузки истории, добавление UI;
- добавление поля с источником свечей (для возможности фильтрации торгов по входным);
- фикс проблемы 70001 ошибки для getOrderState по старым заявкам, инструменты по которым уже удалены.

**Comment by nonamegithub** [comment]
где [возможность получения заданного количества свечей](https://github.com/Tinkoff/investAPI/issues/262)?
